core/src/main/scala/io/finch/syntax/scalafutures.scala

```diff
@@ -1,12 +1,17 @@
-package io.finch.syntax.scala
+package io.finch.syntax
 
-import scala.concurrent.{Future => ScalaFuture}
+import com.twitter.util.{Future, Promise}
+import scala.concurrent.{ExecutionContext, Future => ScalaFuture}
 import scala.util.{Failure, Success}
 
-import com.twitter.util.{Future, Promise}
-import io.finch.syntax.ToTwitterFuture
+object scalafutures extends EndpointMappers {
```

Comment:
As it's no longer a package object, can we rename it to `scalaFutures` instead?

Reply:
Will do.